mcs/class/corlib/corert/ReflectionAugments.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Reflection;
+
+namespace Internal.Reflection.Augments {
```

Comment:
We don't normally use the `Internal` namespace in Mono.  Also i'm not sure what this has to do with `corert`

Reply:
If we don't want to bring the whole classes from CoreRT we just put some pieces to `corert` preserving original namespace. E.g. the same is done in `corert/RuntimeAugments.cs`.